parsec/core/fs/workspacefs/sync_transactions.py

```diff
@@ -415,8 +415,16 @@ async def file_conflict(
                 await self.local_storage.set_manifest(parent_id, new_parent_manifest)
                 await self.local_storage.set_manifest(entry_id, other_manifest)
 
-                self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=new_manifest.id)
-                self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent_id)
+                self._send_event(
+                    CoreEvent.FS_ENTRY_UPDATED,
```

Comment:
same as above: we shouldn't use the same reason for the conflicting entry and it parent (otherwise it would become harder to determine what to do with this event from a listener point of view)

So we should have `FSEntryUpdatedReason.SYNC_FILE_CONFLICT_ENTRY_CREATION` and `FSEntryUpdatedReason.SYNC_FILE_CONFLICT` (which would expose `entry_id` and `entry_name`)